packages/frameworks-sveltekit/.env

```diff
@@ -0,0 +1,4 @@
+# These environment variables are used by Sveltekit to inject into `$env/static/private`
```

Comment:
Why is this file added to the package itself?

Reply:
Sorry, I used TS `declare` instead 😅